pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -731,11 +694,55 @@ func Test_EnforceEnvironmentVariablePolicy_NarrowingMatches(t *testing.T) {
 	}
 }
 
+func Test_WorkingDirectoryPolicy_Matches(t *testing.T) {
+	testFunc := func(gc *generatedContainers) bool {
+		tc, err := setupContainerWithOverlay(gc, true)
+		if err != nil {
+			t.Error(err)
+			return false
+		}
+
+		if err := tc.policy.EnforceOverlayMountPolicy(tc.containerID, tc.layers); err != nil {
+			t.Errorf("failed to enforce overlay mount policy: %s", err)
+			return false
+		}
+
+		err = tc.policy.enforceWorkingDirPolicy(tc.containerID, tc.container.WorkingDir)
+		return err == nil
```

Comment:
Could be 
```go 
return tc.policy.enforceWorkingDirPolicy(tc.containerID, tc.container.WorkingDir) == nil
``

Reply:
Or was this because things were getting a little far off to the right? Realized on my monitor this doesn't look half bad but on my laptop it's a little squished with my suggestion

Reply:
yeah, it could be. the line isn't that long. just didn't want it to look too "busy"